universum/modules/error_state.py

```diff
@@ -36,25 +36,24 @@ def check_required_option(self, setting_name: str, message: str) -> bool:
             return False
         return True
 
-    def read_and_check_multiline_option(self, setting_name: str, error_message: str) -> str:
-        try:
-            value: str = getattr(self.settings, setting_name, None)
-            if value.startswith('@'):
-                try:
-                    with open(value.lstrip('@')) as file_name:
-                        result = file_name.read()
-                except FileNotFoundError as e:
-                    self.error(f"Error reading argument {setting_name} from file {e.filename}: no such file")
-                    return ""
-
-            elif value == '-':
-                result = "".join(sys.stdin.readlines())
-            else:
-                result = value
-        except AttributeError as e:
-            self.error(error_message)
+    def read_multiline_option(self, setting_name: str) -> str:
+        value: str = getattr(self.settings, setting_name, None)
+        if not value:
             return ""
+        if value.startswith('@'):
+            try:
+                with open(value.lstrip('@')) as file_name:
```

Comment:
Please rename file_name to file

Reply:
`file` is a reserved name that shouldn't be used as a variable name in Python

Reply:
Then invent some other name.
file_name is definitely misleading.